eta/detectors/tfmodels_detectors.py

```diff
@@ -385,6 +382,157 @@ def _evaluate(self, imgs, ops):
         return self._sess.run(out_tensors, feed_dict={in_tensor: imgs})
 
 
+class TF2ModelsDetectorConfig(Config, etal.HasPublishedModel):
+    """TF2ModelsDetector configuration settings.
+
+    Note that `labels_path` is passed through
+    `eta.core.utils.fill_config_patterns` at load time, so it can contain
+    patterns to be resolved.
+
+    Attributes:
+        model_name: the name of the published model to load. If this value is
+            provided, `model_path` does not need to be
+        model_path: the path to a frozen inference graph to load. If this value
```

Comment:
I think this now must be an `archive` of a `SavedModel` directory or something like that

Reply:
Right. On a different note, is there something that I need to do with `model_name` or is does it automatically populate  `model_path` if it is provided instead?

Reply:
`model_name` and `model_path` are defined by the `eta.core.learning.HasPublishedModel` interface. When adding models to `eta/models/manifest.json`, you should populate `model_name` and not `model_path` in the default deployment config for the model. When someone loads the model via `etal.load_default_deployment_model()`, then the model's weights will be downloaded and `model_path` will be populated at runtime.

You'll notice that in FiftyOne, and model `manifest.json` files don't include `model_name` or `model_path` in the default deployment config. The appropriate value (for `model_path` in this case, not `model_name`, since we want to let FiftyOne control where the models are downloaded, not ETA) is inserted by `load_zoo_model()` when someone loads a model.